### PR TITLE
Remove unnecessary whitespace from tables

### PR DESCRIPTION
**Pull request by TPGPL**
- `70d949d`
  - ` {2,}\|` -> ` |`
  - `\| {2,}` -> `| `
- `0a573a5`
  - `\|.* {2,}.*\| ` to seek unnecessary whitespace + minor regexes to fix various cases

*blocked till i look through all files and check for potential errors*

### CODE REVIEW

File: news/2018-02-22-osu-mania-7k-world-cup-ro16-recap.md [hunk 1]
```diff
@@ -95,21 +95,21 @@ That concludes all the matches for this weekend! Good luck to all the remaining
 
 ### Saturday, 24. February 2018
 
-| Team A |  |  | Team B | Match Time (UTC) |
+| Team A | | | Team B | Match Time (UTC) |
```
Comment (MegaApplePi) [inline news/2018-02-22-osu-mania-7k-world-cup-ro16-recap.md:98]: Empty cells need two spaces
```suggestion
| Team A |  |  | Team B | Match Time (UTC) |
```

**Comment by cl8n** [comment]
this is the kind of thing that should be left to a markdown parser since the structure is identical and only syntax changes

**Comment by peppy** [comment]
can this not be done by running remark locally?

**Comment by cl8n** [comment]
`remark -o` over the whole wiki isn't possible yet without some annoying side effects, so we've been avoiding it. fixing remark-stringify isn't a far goal though, I don't see reason for pure syntax changes anymore when soon I'll be able to trust remark to do it perfectly

**Comment by cl8n** [comment]
fwiw @TPGPL 's other PRs are helpful in being able to "trust" `remark -o` as well because they usually fix syntax that's completely broken or unintended

**Pull request merged**